sparfa_server/loaders.py

```diff
@@ -182,7 +169,8 @@ def load_course(course_uuid, cur_sequence_offset = None, sequence_step_size=1):
                                                      cur_sequence_offset)
 
         cur_sequence_offset = handle_course(cur_event_data,
-                                            sequence_step_size=sequence_step_size)
+                                            sequence_step_size = sequence_step_size,
```

Comment:
This is a stylistic nitpick. There should be no spaces around the equal signs for parameters.